calico-enterprise/reference/installation/helm_customization.mdx

```diff
@@ -0,0 +1,129 @@
+
+```yaml
+installation:
+  enabled: true
+  <installation resource spec fields that user wants to configure>
+
+apiServer:
+  enabled: true
+  <apiServer resource spec fields that user wants to configure>
+
+intrusionDetection:
+  enabled: true
+  <intrusionDetection resource spec fields that user wants to configure>
+
+logCollector:
+  enabled: true
+  <logCollector resource spec fields that user wants to configure>
+
+logStorage:
+  enabled: true
+  nodes:
+    count: 1
+  <logStorage resource spec fields that user wants to configure>
+
+manager:
+  enabled: true
+  <manager resource spec fields that user wants to configure>
+
+monitor:
+  enabled: true
+  <monitor resource spec fields that user wants to configure>
+
+compliance:
+  enabled: true
+  <compliance resource spec fields that user wants to configure>
+
+policyRecommendation:
+  enabled: true
+  <policyRecommendation resource spec fields that user wants to configure>
+
+authentication:
+  enabled: false
+  <if enabled is true: authentication resource spec fields that user wants to configure>
+
+applicationLayer:
+  enabled: false
+  <if enabled is true: applicationLayer resource spec fields that user wants to configure>
+
+amazonCloudIntegration:
+  enabled: false
+  <if enabled is true: amazonCloudIntegration resource spec fields that user wants to configure>
+
+defaultFelixConfiguration:
+  enabled: false
+  <if enables is true: felixConfiguration resource spec fields that user wants to configure>
+```
+
+## Common customizations
+
+Common customizations that you might want to configure are number of replicas, pod affinity, and encryption using WireGuard.
+
+### Number of replicas
+This setting defines the number of replicas for {{prodname}} components that can run simultaneously in multiple instances.
+To configure this setting, see [controlPlaneReplicas](api.mdx#operator.tigera.io/v1.InstallationSpec).
+The components for the replicas are:
+
+- tigera-manager
+- tigera-apiserver
+- tigera-dex
+- tigera-kibana
+- calico-node
+
+To set a specific replica for these components, you must provide the setting in your custom values.yaml file:
+```yaml
+installation:
+  enable: true
+  controlPlaneReplicas: 3 # desired number of replicas for the components listed above.
+```
+
+### Pod affinity
+You can define pod affinity for the following Tigera components. Update the appropriate custom resource in your custom `values.yaml`.
+
+- tigera-apiserver: through ApiServer resource.
+- calico-nodes: through CalicoNodeDaemonSet property in the Installation resource.
+- calico-kubecontroller: through CalicoKubeControllersDeployment property in the Installation resource.
+- compliance deployment pods (compliance-snapshotter, compliance-server, compliance-controller, compliance-benchmarker,
+compliance-scaleloader, compliance-reporter): through Compliance resource.
+- pods created for the Elasticsearch nodes: through LogStorage resource.
```

Comment:
Could we update the description so that it more accurately represents the current LogStorage Spec docs?